### DISCUSSION TITLE
Request: show time per state instead of state count in pie chart in Current State dashboard

### DISCUSSION THREAD

**Discussion by sjvc** [jheredianet/Teslamate-CustomGrafanaDashboards#30] [Ideas]
The pie chart in the "Current State" dashboard is showing the number of times the car entered in each state. I think it will be more useful to see how much time the car has been in each state. 

Also, I find the stacked bar chart at the bottom of the "Overview" dashboard that comes with the TeslaMate very helpful. So I think it would be great if you could also add it to your "Current State" panel (below the pie chart for example), so that the information about the states is as complete as possible.

Thank you.

**Comment by jheredianet** [comment]
Good point. I'll make the changes and let you know when it's done. 
Cheers!

**Comment by jheredianet** [comment]
Hey! You got it. Check it out the new realase
https://github.com/jheredianet/Teslamate-CustomGrafanaDashboards/releases/tag/v2.8.3

**Comment by sjvc** [comment]
Thank you!!
